cli/dcoscli/tables.py

```diff
@@ -425,13 +425,35 @@ def pod_table(pods):
     :rtype: PrettyTable
     """
 
+    def id_and_containers(pod):
+        """Extract the pod ID and container names from the given pod JSON.
+
+        :param pod: the pod JSON to read
+        :type pod: {}
+        :returns: the entry for the ID+CONTAINER column of the pod table
+        :rtype: str
+        """
+
+        pod_id = pod['id']
+        container_names = sorted(container['name'] for container
+                                 in pod['spec']['containers'])
```

Comment:
are these fields <i>always</i> going to appear in the returned json? Same questions with fields in OrderedDict below